stable/aws-node-termination-handler-chart/values.yaml

```diff
@@ -0,0 +1,24 @@
+# name is the main project keyword. All pods, cluster roles,
+# service accounts, etc. are created under this name.
+name: "node-termination-handler"
+
+namespace: "kube-system"
+
+# image values
+imageName: "amazon/aws-node-termination-handler"
+imageVersion: "v1.0.0"
+
+# deleteLocalData tells kubectl to continue even if there are pods using
+# emptyDir (local data that will be deleted when the node is drained).
+deleteLocalData: \"false\"
```

Comment:
here and below for ignoreDaemonSets and gracePeriod, you don't need the backslash escapes.